features/rgb.yml

```
@@ -0,0 +1,14 @@
+name: RGB
+description: The `rgb()`, rgba()` and hexadecimal notation pick colors using red, green and blue channels.
```

Nits for serial comma, a typo, and parallelism. I think we also ought to mention the alpha channel, if we're pinning the headline support status to it:

```suggestion
description: The `rgb()`, rgba()`, and hexadecimal (as in `#004488ff`) notations pick colors using red, green, blue, and alpha (transparency) channels.
```